workflow-to-js-transformer/src/main/java/com/github/cafdataprocessing/workflow/transform/TransformerFunctions.java

```diff
@@ -15,13 +15,23 @@
  */
 package com.github.cafdataprocessing.workflow.transform;
 
+import com.github.cafdataprocessing.processing.service.client.ApiClient;
+import com.github.cafdataprocessing.processing.service.client.ApiException;
+import com.github.cafdataprocessing.processing.service.client.api.TenantsConfigurationApi;
```

Comment:
Change to TenantConfigurationApi as the generated library class changed name recently.